README.md

```diff
@@ -14,70 +19,43 @@ Read on to get started
 	```xml
 	<dependency>
 		<groupId>io.swagger.assert</groupId>
-		<artifactId>assert4j-facade</artifactId>
+		<artifactId>swagger-assert4j-facade</artifactId>
 		<version>1.0.0-SNAPSHOT</version>
 	</dependency>
 	```
 
 2. Create and execute a simple recipe with your favorite unit-testing framework:
 
 	```java
-	    @Test
-        public void simpleCountTest() throws Exception {
-             RecipeExecutionResult result = executeRecipe(
-                 GET("https://api.swaggerhub.com/specs")
-                     .withParameters(
-                         query( "specType", "API" ),
-                         query( "query", "testserver" )
-                     )
-                     .withAssertions(
-                         json("$.totalCount", "4")
-                     )                 
-                 );
-    
-            assertExecutionResult(result);
-        }
+    @Test
+    public void simpleCountTest() throws Exception {
+         RecipeExecutionResult result = executeRecipe("Simple Count Test",

```

Comment:
I understand that you want to label the recipe in this way, to comment on what it's doing. However, this makes the API look more complex than it is, so I'd prefer if we didn't pass the parameter.